### PR TITLE
RootCA Key Usage Critical Lint

### PR DESCRIPTION
**Pull request by kumarde**

### CODE REVIEW

File: lints/lint_root_ca_key_usage_must_be_critical.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	// Internal data here
+}
+
+func (l *rootCAKeyUsageMustBeCritical) Initialize() error {
+	return nil
+}
+
+func (l *rootCAKeyUsageMustBeCritical) CheckApplies(c *x509.Certificate) bool {
+	// Add conditions for application here
+	fmt.Println(c.SelfSigned)
```
Comment (zakird) [inline lints/lint_root_ca_key_usage_must_be_critical.go]: 👿

Comment (kumarde) [inline lints/lint_root_ca_key_usage_must_be_critical.go]: fml

**Comment by kumarde** [comment]
@zakird it's better now, i swear!

File: lints/lint_root_ca_key_usage_must_be_critical.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+type rootCAKeyUsageMustBeCritical struct {
+	// Internal data here
+}
+
+func (l *rootCAKeyUsageMustBeCritical) Initialize() error {
+	return nil
+}
+
+func (l *rootCAKeyUsageMustBeCritical) CheckApplies(c *x509.Certificate) bool {
+	// Add conditions for application here
```
Comment (zakird) [inline lints/lint_root_ca_key_usage_must_be_critical.go]: doesn't really seem like you need this comment either.

**Pull request merged**